examples/jdbc-atomic-transaction/jdbc_atomic_transaction.md

```diff
@@ -17,3 +15,7 @@ The following util files will initialize the test database before running the BB
 ::: code initialize.bal :::
 
 ::: code cleanup.bal :::
+
+## Related Links
+- [`jdbc:Client` - API documentation](https://lib.ballerina.io/ballerinax/java.jdbc/latest/).
+- [`jdbc:Client` - Specification](https://github.com/ballerina-platform/module-ballerinax-java.jdbc/blob/master/docs/spec/spec.md).
```

Comment:
This should point to the exact part of the spec, right?

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-distribution/pull/3573